ruby/private/bundle.bzl

```diff
@@ -33,19 +33,25 @@ def _rb_bundle_impl(repository_ctx):
 
     env = {
         "BUNDLE_BIN": repr(binstubs_path),
-        "BUNDLE_GEMFILE": gemfile.basename,
+        "BUNDLE_GEMFILE": repr(gemfile_path),
```

Comment:
actually i think this change is not ideal --  it's currently using the abs path of the source Gemfile. will make this more hermetic by copying the Gemfiles into the bundle repository so we don't have to depend on source during `bundle install` and `bundle exec`

Reply:
ended up not pursuing this for now, it's a bit tricky with local gems, since it would mean all the dependent .rb files will also have to be added to the `srcs` attribute in `rb_bundle`